fala/settings/base.py

```diff
@@ -142,6 +142,8 @@ def root(*x):
 
 LAALAA_API_HOST = os.environ.get("LAALAA_API_HOST", None)
 
+ENVIRONMENT = os.environ.get("ENVIRONMENT", "dev")
```

Comment:
I would prefer to change this to "unknown" or something. It makes me nervous to assume dev, especially as dev is going to be the environment we're least cautious if we think we're in

Reply:
If this is changed to unknown, how we will set up an environment for dev? Should I assume an unknown environment is equal to development? Other reasoning being this is the same set up as on cla_public

Reply:
> Should I assume an unknown environment is equal to development?

No, that's exactly the thing I said we shouldn't do.
If this is how it's been done on cla_public it needs fixing on cla_public

If we're going to mark an environment as "dev" there has to be _something_ explicitly stating that (ie setting the env var to that value) and not just us identifying the lack of it being set. As a thought experience: if I inadvertently delete the line from our kubernetes setup that sets the env var to "production" I *do not* want it to start saying "dev" on the site: 1) it breaks the intended experience, 2) it masks the problem that we unexpectedly don't know which environment it is

As to how to solve it, for one idea (may or may not be the best one) have a think back to this morning and how we have a `local_settings.py` file (which we were looking at an API_HOSTNAME setting in, but can be used for any setting).